### DISCUSSION TITLE
Hide SPECIFIC Models Option Instead of All or Nothing

### DISCUSSION THREAD

**Discussion by MaladyMawr** [civitai/civitai#155] [Ideas]
Hello, as someone who doesn't want to mute all NSFW content/all of a creators models, as some models are very helpful due to the use of anatomy for creating images, I think it would be nice to have a 3rd hide option. I don't want to keep having to browse past the sea of blow job models, or even in non-NSFW the 3rd, 4th etc Jenna Ortega embed/ckpt.

I don't always want to hide all NSFW content, or hide all content from a creator. But Individual models/embeds. Having this option could be in the drop down right next to the hide all content from this creator option. It would help to keep browsing cleaner.

Thanks for consideration

**Comment by JustMaier** [comment]
This makes sense. I'll get this on our todo list

**Comment by JustMaier** [comment]
**This is live!**
![image](https://user-images.githubusercontent.com/607609/213083602-c8523a05-3ad7-47d6-84eb-1fc707f65993.png)
You can now hide any model from your feed.

You can also review any model that you've hidden by visiting:
`https://civitai.com/?hidden=true`

They can be unhidden the same way or from the model's page from the context menu.

**Reply by MaladyMawr** [reply]
awesome!! thank you!

**Reply by MaladyMawr** [reply]
When hiding models after a certain number of them it suddenly refuses to load anything, if I unhide 1 model from my total it shows everything normally. (and yes i triple checked all the normal filters etc)
<img width="911" alt="Capture" src="https://user-images.githubusercontent.com/122900563/213209430-be0ae8c2-1baa-493b-b39d-e58c68d5f673.PNG">
